README.md

```diff
@@ -37,16 +37,14 @@ e.g. [direnv](https://direnv.net).
 
 The values can be found on [Confluence](https://tools.hmcts.net/confluence/x/eQP3P).
 
-Run the final script each time new LA users are added in order to fix access segregation locally
-
 ```bash
 $ ./bin/configurer/add-services.sh
 $ ./bin/configurer/add-roles.sh
 $ ./bin/configurer/add-users.sh
-$ ./bin/configurer/utils/generate-local-user-mappings.sh
-
 ```
 
+Users are defined in `bin/configurer/users.json`. Run the final script each time new LA users are added in order to fix access segregation locally.
```

Comment:
What do you mean by final script? You removed `./bin/configurer/utils/generate-local-user-mappings.sh` from list of scripts above.

Reply:
Alright, I saw you moved it into `./bin/configurer/add-users.sh`